libraries/rush-sdk/package.json

```diff
@@ -44,7 +44,8 @@
     "@rushstack/node-core-library": "workspace:*",
     "@rushstack/package-deps-hash": "workspace:*",
     "@rushstack/terminal": "workspace:*",
-    "tapable": "2.2.1"
+    "tapable": "2.2.1",
+    "true-case-path": "~2.2.1"
```

Comment:
I did my best to murder usage of this package because its operation is very expensive; why is it back?